bot/index.js

```diff
@@ -34,47 +52,28 @@ client.on('connected', onConnectedHandler);
 // Connect to Twitch:
 client.connect();
 
-// Called every time a message comes in
-function onMessageHandler (target, context, msg, self) {
-  if (self) { return; } // Ignore messages from the bot
-
-  // Remove whitespace from chat message
-  const commandName = msg.trim();
-
-  if (commandName[0] != '!' && commandName[0] != 'o')
-    return;
-  console.log(`Parsing message: ${msg}`);
-  
-  const commandArray = commandName.split(' ');
-  const cmd = commands.getCommand(commandArray[0], context.username);
-
-  if (cmd) {
-      cmd(client, target, context, commandArray.slice(1), coachBot.onMessage);
-  } else {
-      console.log(`${commandName}: is borked`);
-  }
-}
-
-// Called every time the bot connects to Twitch chat
-function onConnectedHandler (addr, port) {
-    console.log(`* Connected to ${addr}:${port}`);
-}
+const ANON_GIFT_PAID_UPGRADE = 'anongiftpaidupgrade';
+const GIFT_PAID_UPGRADE = 'giftpaidupgrade';
+const RESUB = 'resub';
+const SUB_GIFT = 'subgift';
+const SUB_MYSTERY_GIFT = 'submysterygift';
+const ANON_SUB_GIFT = 'anonsubgift';
+const ANON_SUB_MYSTERY_GIFT = 'anonsubmysterygift';
+const PRIME_PAID_UPGRADE = 'primepaidupgrade';
+const SUBSCRIPTION = 'subscription';
 
 const coachBot = {
-  
-}
+    [ANON_GIFT_PAID_UPGRADE]: () => eventLogger(ANON_GIFT_PAID_UPGRADE),
+    [GIFT_PAID_UPGRADE]: () => eventLogger('giftpaidupgrade', subEvents.newSubscriber),
+    [RESUB]: () => eventLogger('resub', subEvents.newSubscriber),
+    [SUB_GIFT]: () => eventLogger('subgift', subEvents.newSubscriber),
+    [SUB_MYSTERY_GIFT]: () => eventLogger('submysterygift', subEvents.newSubscriber),
+    [ANON_SUB_GIFT]: () => eventLogger('anonsubgift', subEvents.newSubscriber),
+    [ANON_SUB_MYSTERY_GIFT]: () => eventLogger('anonsubmysterygift', subEvents.newSubscriber),
+    [PRIME_PAID_UPGRADE]: () => eventLogger('primepaidupgrade', subEvents.newSubscriber),
+    [SUBSCRIPTION]: (channel, username, method, message, userstate) => eventLogger('subscription', () => subEvents.newSubscriber(userName, coachBot.onMessage))
```

Comment:
You don't have to put the lambdas in the object literal, either. You can just as easily assign the functions to variables outside of this literal and *then* assign it as a value. Up to you.